dynamic_analysis_pkg/dynamic_analysis_pkg/__init__.py

```diff
@@ -0,0 +1,167 @@
+# pylint: disable=missing-function-docstring
+'''Machinary to fetch simulation results to a Jupyter Notebook'''
+from pathlib import Path, PurePath
+from shutil import move
+from datetime import datetime
+import logging
+import os
+import pandas as pd
+from tabulate import tabulate
+import pyunicore.client as unicore_client
+from tqdm.notebook import tqdm
+
+L = logging.getLogger('dynamic-analysis-pkg')
+L.setLevel(logging.INFO)
+
+UNICORE_ENDPOINT = 'https://bspsa.cineca.it/advanced/pizdaint/rest/core'
+
+
+class AResults:
+    '''
+    Based on a simulation id, fetch the results using Unicore API
+    '''
+    DEFAULT_CIRCUIT_DIR = '/home/data-bbp/20191017/'
+    DEFAULT_WD_BASE = '/home/simulation-results'
+    ORIGIN_CIRCUIT_PATH = '/store/hbp/ich002/antonel/O1/20191017/'
+    HPC_DIR_BASE = '/scratch/snx3000/unicore/FILESPACE/'
+    JOB_BASE = os.path.join(UNICORE_ENDPOINT, 'jobs')
```

Comment:
Define all those as `Path`. It will make most of the following `Path` calls unnecessary.